jenkins/bootstrap_test.py

```diff
@@ -1980,6 +1980,8 @@ def testValidJobEnvs(self):
         with open(bootstrap.test_infra('jobs/config.json')) as fp:
             config = json.loads(fp.read())
             for job in config:
+                self.assertTrue('sigOwners' in config[job], job)
+                self.assertTrue(type(config[job]['sigOwners']) == list)
```

Comment:
Why can't we treat empty as unowned, and once they are owned we extend the test to require non-empty?